README.md

```diff
@@ -110,3 +113,32 @@ RULES = [
     ),
 ]
 ```
+
+## Container Usage ##
```

Comment:
Let's call its "Docker", it's not about a common container technology, it's docker-specific details. 

I've created a repository for the project `devopshq/artifactory-cleanup` - https://hub.docker.com/r/devopshq/artifactory-cleanup and I'll push an image after we finish the PR or add github actions that will deploy it, so no needs in "how to build" docker section